spec/actors/hyrax/actors/interpret_visibility_actor_spec.rb

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Remove this blank line please.

Reply:
I added a line above here, so I think it's okay now.